Containerfile

```diff
@@ -17,33 +18,29 @@ COPY ./scripts ./scripts
 RUN dnf install -y graphviz python-devel pcre-devel
 RUN pip install tox
 
-RUN if [ -z "${RHOSO_DOCS_GIT_URL}" ]; then \
+RUN if [ "$BUILD_UPSTREAM_DOCS" = "true" ]; then \
         ./scripts/get_openstack_plaintext_docs.sh; \
     fi
 
 RUN if [ ! -z "${RHOSO_DOCS_GIT_URL}" ]; then \
-        ./scripts/get_rhoso_plaintext_docs.sh; \
+        OUTPUT_DIR_NAME=rhoso-docs-plaintext ./scripts/get_rhoso_plaintext_docs.sh; \
```

Comment:
(non-blocking) I think we could eliminate OUTPUT_DIR_NAME here and change the default in the script? (See below)

Reply:
Good point, yeah, I was a bit confused when I generated the RHOSO and it used the exact same name as the upstream docs. I will do that